content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst

```diff
@@ -0,0 +1,289 @@
+newest instruction titled `New Step Suggestion:`, followed by the user-entered title. Click the line
+item to view the suggested changes.
+
+.. image:: engineering_change_orders/show-instructions.png
+   :align: center
+   :alt: "Show Instructions" icon in the *Operations* tab of a BoM.
+
+On the :ref:`quality control point <quality/quality_control_points>` form, ensure the following form
+fields are accurately filled out to give detailed instructions for operators:
+
+- :guilabel:`Title`: rename to give a concise description of the new instruction
+- :guilabel:`Control per`: using the drop-down menu, determine whether this instruction applies
+  broadly for the :guilabel:`Product`, specifically for this :guilabel:`Operation` *only*, or a
+  particular :guilabel:`Quantity` of the product
+- :guilabel:`Type`: categorizes the control point type. From the drop-down menu, select
+  :guilabel:`Instructions` to detail an instruction for the worker. To receive input from the
+  workers, select the :guilabel:`Take a Picture`, :guilabel:`Register Consumed Materials`,
+  :guilabel:`Print Label`, or other :ref:`quality check options <quality/quality_control_points>`.
+
+Once the quality control point is configured, return to the :guilabel:`Steps` list using the
+breadcrumbs. Finally, drag the last quality control line item to its intended order of instructions.
+
+.. example::
+   Drag and reorder the `Check for broken switches` instruction, by clicking and dragging its "6
+   dots" icon to move it from the bottom to the second position.
+
+   .. image:: engineering_change_orders/reorder.png
+      :align: center
+      :alt: Drag and reorder instructions by selecting the "6 dots" icon on the far left.
+
+.. _plm/eco/eco-type:
+
+Create ECO type
+===============
+
+To access and manage ECO types, navigate to :menuselection:`PLM app --> Configuration --> ECO
+Types`.
+
+Create a new ECO type by clicking :guilabel:`New`, and fill in the :guilabel:`Name`. The
+:guilabel:`Email Alias` field is optional, but feedback submitted to the filled-in email address
+will automatically create an |ECO| in this ECO type.
+
+Modify existing ECO type names and email aliases by clicking on the respective ECO type in the list.
+
+Existing ECO types are displayed in the task form, on the :guilabel:`PLM Overview` page, which is
+accessible by navigating to :menuselection:`PLM app --> Overview`.
+
+.. _plm/eco/stage-config:
+
+Stage configuration
+-------------------
+
+Click an ECO type from :menuselection:`PLM app --> Overview` to open a kanban view of |ECOs| of this
+type.
+
+In the pipeline, ensure there is at least one *verification* stage, where |ECO| changes can be
+applied, and a *closing stage*, for storing completed |ECOs|.
+
+To configure a stage, hover over the intended stage, and select the :guilabel:`⚙️ (gear)` icon.
+Then, click :guilabel:`Edit` to open a pop-up window.
+
```

Comment:
a screenshot of the Edit: New pop-up window that appears when the gear icon is clicked would be useful